src/app/accounts/register/register.component.html

```diff
@@ -10,23 +10,51 @@ <h2 class="tui-form__header">Register</h2>
                     <div class="tui-form__row">
                         <tui-input aria-label="Email" formControlName="email">Email</tui-input>
                         <tui-field-error formControlName="email"></tui-field-error>
-                        <div class="tui-form__field-note">We'll never share your email with anyone else.</div>
+                        <div class="tui-form__field-note">We'll never share your email with anyone
+                            else.
+                        </div>
```

Comment:
I think this is false, we share them with the prof, right? Can we update this?

Reply:
I am not entirely sure. For the grade book, I think it should just be first name and last name, but I am not entirely sure about that.

Reply:
I can still update this if you would like though.

Reply:
I just removed it because I am not entirely certain, and would prefer to be careful.